app/notifications/rest.py

```diff
@@ -181,9 +181,10 @@ def _service_allowed_to_send_to(notification, service):
         if api_user.key_type == KEY_TYPE_TEAM:
             message = 'Can’t send to this recipient using a team-only API key'
         else:
+            # FIXME: hard code it for now until we can get en/fr specific links and text
```

Comment:
Wondering if we're supposed to write "hard code" or "hardcode"

Reply:
ha! good question, I've looked online and they are both accepted 👍